src/epic.js

```diff
@@ -0,0 +1,83 @@
+import Q from 'q';
+import {
+  sendRequest,
+  validateDate,
+} from './util';
+
+export default function epic(): object {
+  function validateType(type: string): string {
+    const validTypes = ['natural', 'enhanced'];
+    const i = validTypes.indexOf(type.toLowerCase());
+    return i === -1 ? '' : validTypes[i];
+  }
+
+  function badRequest(message: string, deferred: object): undefined {
```

Comment:
Got it. Good thinking on the renaming, I'll get those changes done in the next commit